test-tool/test_write_residuals.c

```diff
@@ -82,9 +83,9 @@ write_residuals_test(const struct residuals_test_data *tdata)
         memset(task, 0, sizeof(*task));
 
         if (tdata->check_overwrite) {
-                memset(scratch, 'b', (tdata->xfer_len * tdata->buf_len));
+                memset(scratch, 'b', tdata->buf_len);
         } else {
-                memset(scratch, 0xa6, (tdata->xfer_len * tdata->buf_len));
+                memset(scratch, 0xa6, tdata->buf_len);
```

Comment:
Are these changes perhaps a bug fix for the previous patch in this series? If so, please move these into the previous patch.

Reply:
done